livechatsnew/views.py

```diff
@@ -0,0 +1,84 @@
+import os
+from datetime import datetime
+
+from bson import ObjectId
+from django.shortcuts import redirect, reverse
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from pymongo import MongoClient
+from pydantic import ValidationError
+
+from authentication.models import CustomUser
+from .schemas import Conversation
+
+client = MongoClient(os.environ.get("MONGODB_HOST"))
+collections = client.livechats.conversations
+
+
+# Create your views here.
+@api_view(['POST'])
+def start_conversation(request, ):
+    """ Creating conversation, if it doesn`t exists, after this pk or conversation ll be used in web socket link"""
+    data = request.data
+    user_email = data.pop("email")
```

Comment:
Using ```pop``` will remove "email" from data. If data should be used later, this might lead to unexpected behavior. Use ```data.get("email")``` if you don't intend to modify data